classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students.push(student)
+    student.classroom = self
+  end
```

Comment:
[OPTIONAL] You can create a condition that will prevent a student from being added to a classroom if the student already exists in the class. Logically, a student cannot exist in the same classroom twice

Another approach to this method would be:
```
  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```

- This method serves as the 'belongs-to' side (a student belongs to a classroom) as well as prevents a student from being added to a classroom if he/she already exists in the class.